examples/microsoft-azure-monitor-logs.yml

```diff
@@ -0,0 +1,11 @@
+integrations:
+  - name: nri-flex # We're telling the Infra agent to run Flex!
+    config: # Flex configuration starts here!
+      name: msAzureMonitorLogs
+      apis:
+        # Event Requests
+        - event_type: msAzureMonitorStatus # Name of the event in New Relic
+          commands:
+            #- run: /etc/newrelic-infra/integrations.d/exec.sh
+            - run: dotnet /etc/newrelic-infra/integrations.d/ms_azure_monitor_event_hubs_receiver.dll <NEWRELIC_LICENSE_KEY>
+              timeout: 59000
```

Comment:
the default 30s interval is still applied at the Integrations SDK level here; is a 59s timeout going to cause conflicts?

Reply:
I don't think so